ansible/vars/icds/icds_public.yml

```diff
@@ -61,6 +61,9 @@ postgres_users: "{{ secrets.POSTGRES_USERS }}"
 
 postgres_config:
   max_connections: 300
+  work_mem: '8MB'
+  shared_buffers: '512MB'
```

Comment:
We increased from 128MB to 512MB, I want to increase and test a few times before we go as high as recommended, we can try 2GB next and 8GB... but not if the system is not actually taking advantage of it.